### PR TITLE
v2 stationboard tooltip

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by pfanguin** [comment]
test link : https://mf-geoadmin3.dev.bgdi.ch/fap_oev/?topic=ech&lang=fr&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.bav.haltestellen-oev&X=151100.54&Y=538088.16&zoom=8

**Comment by pfanguin** [comment]
jenkins test

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -3315,7 +3369,7 @@ msgid "ch.bav.haltestellen-oev.name"
 msgstr "Name"
 
 msgid "ch.bav.haltestellen-oev.next_departures"
-msgstr "Next departures from"
+msgstr "next departures to"
```
Comment (loicgasser) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: capital letter for next?

File: chsdi/templates/htmlpopup/oev_haltestellen.mako [hunk 2]
```diff
@@ -56,11 +80,12 @@ $(document).ready(function() {
           var then = result2[i].departureDate;
           var diff = moment(then,"DD/MM/YYYY HH:mm").diff(moment(now,"DD/MM/YYYY HH:mm"));
           var d = moment.duration(diff);
-          var s = Math.floor(d.asHours()) + moment.utc(diff).format(":mm");
-          numero += '<p style="height:24px; margin:0px;">' + result2[i].label + '</p>';
-          destination += '<p style="height:24px; margin:0px;">' + result2[i].destinationName + '</p>';
-          departures += '<p style="height:24px; margin:0px;">' + result2[i].time + '</p>';
-          timeDiff += '<p style="height:24px; margin:0px;"><b>' + s +'</b></p>';
+          var s = Math.floor(d.asMinutes()) + moment.utc(diff).format("[']", -1);
+          console.log(s);
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/oev_haltestellen.mako]: console

File: chsdi/templates/htmlpopup/oev_haltestellen.mako [hunk 3]
```diff
@@ -16,6 +16,30 @@
...
+    vertical-align: middle !important;
+  } 
+
+  #label{
+    border: 1px solid #D8D8D8;
+    border-radius: 4px;
+    width: 30px;
+  }
+  
+  #label, #destination, #time, #time-diff {
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/oev_haltestellen.mako]: could you use classes instead?

**Comment by loicgasser** [comment]
I like the new tooltip, nice imrpovement.

File: chsdi/locale/en/LC_MESSAGES/chsdi.po (hunk 1 above)
Comment (pfanguin) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: yes, it's correct no capital, because it's not the beginning of the sentence.

Comment (loicgasser) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: k

**Comment by loicgasser** [comment]
jenkins test

**Comment by loicgasser** [comment]
fishy

What are those timetables? Code or data?
This dataset is quite weird, also to métro stop is called once:
`Lausanne, Gare`  -> no data
`Lausanne. gare` -> some data but again super weird

**Comment by loicgasser** [comment]
ping @AFoletti

**Comment by loicgasser** [comment]
Try `stationboard/stops/8592098?destination=Lausanne%2C%20Ecole%20des%20Métiers`

**Comment by gjn** [comment]
I think this needs rebase.

**Comment by loicgasser** [comment]
I'll do it as she's gone. We need to figure out what's wrong with the data. This is a shame really.

**Comment by loicgasser** [comment]
@ltclm do you know if the tiles have been updated as we now have a new set of stations I believe.

**Comment by loicgasser** [comment]
rebased

**Comment by ltclm** [comment]
yes, @alcapat  has generated a new timestamp.

**Comment by loicgasser** [comment]
ok thanks @ltclm

File: chsdi/locale/empty_chsdi.po [hunk 4]
```diff
@@ -121,7 +121,61 @@ msgstr ""
...
+msgid "<i>haltestellen_bus_tram_zug</i>"
+msgstr ""
+
+msgid "<i>haltestellen_bus_zug</i>"
+msgstr ""
+
+msgid "<i>haltestellen_luftseilbahn</i>"
+msgstr ""
 
+>>>>>>> change format time difference
```
Comment (gjn) [inline chsdi/locale/empty_chsdi.po]: rebase artefact?

**Comment by loicgasser** [comment]
formatting should be hours, minutes, secs not min secs only I'll adapt.

File: chsdi/locale/empty_chsdi.po (hunk 4 above)
Comment (gjn) [inline chsdi/locale/empty_chsdi.po]: ping @loicgasser

**Comment by loicgasser** [comment]
ok

**Comment by cedricmoullet** [comment]
@gjn ready for today's deploy ?

**Comment by loicgasser** [comment]
yes please deploy

**Pull request merged**